Tests/Unit/MSFT_xIISFeatureDelegation.tests.ps1

```diff
@@ -4,14 +4,57 @@ $global:DSCResourceName = 'MSFT_xIISFeatureDelegation'
 #region HEADER
 
 [String] $moduleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
- if ( (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests'))) -or `
-      (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests\TestHelper.psm1'))) )
+$repoSource = (Get-Module -Name $global:DSCModuleName -ListAvailable)
+
+if (($repoSource -ne $null) -and ($repoSource[0].RepositorySourceLocation.Host -eq 'www.powershellgallery.com'))
 {
-    & git @('clone','https://github.com/PowerShell/DscResource.Tests.git',(Join-Path -Path $moduleRoot -ChildPath '\DSCResource.Tests\'))
+    if ( -not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'Tests\DscResourceTestHelper')) )
+    {
+        # Module was installed from the gallery - so user does not need git in order to run tests
+        $choice = read-host "In order to run this test you need to install a helper module, continue with installation? (Y/N)"
+
+        if ($choice -eq "y"){
+            # Install test folders from gallery
+            Save-Module -Name 'DscResourceTestHelper' -Path (Join-Path -Path $moduleRoot -ChildPath 'Tests')
+        }
+    
+        else 
+        {
+            Write-Error "Unable to run tests without the required helper module - Exiting test"
+            return
+        }
+        
+    }
+    Push-Location
```

Comment:
Commenting again as I apparently did it on an old diff.

I think this could be collapsed to
````powershell
$verfolder = Get-ChildItem -Path (Join-Path -Path $moduleRoot -ChildPath '\Tests\DscResourceTestHelper')
````